src/imitation/algorithms/preference_comparisons.py

```diff
@@ -805,7 +822,13 @@ def _train(self, dataset: PreferenceDataset, epoch_multiplier: float = 1.0):
             collate_fn=preference_collate_fn,
         )
         epochs = round(self.epochs * epoch_multiplier)
-        for _ in range(epochs):
+        outer_loop = range(epochs)
+        if epochs > 1:  # Show a progress bar iff we have more than one epoch
```

Comment:
why not just always use tqdm?

Reply:
We could do that I suppose, I think progress bars with a total of one unit look kind of tacky but it's up to you